src/components/contributions-table.jsx

```diff
@@ -114,7 +122,61 @@ class ContributionsTable extends React.Component {
       return 0;
     };
 
+    function parseElectionName(input) {
+      const [ename, ...rest] = input.split('-');
+
+      if (rest.length === 1) {
+        // If there is only one element after the split, assume it's the year
+        return { ename, month: 'november', year: rest[0] };
+      } else if (rest.length >= 2) {
+        // If there are two or more elements, assume the last is the year
+        // and the second-to-last is the month
+        return { ename, month: rest[rest.length - 2], year: rest[rest.length - 1] };
+      }
+      // Handle the case where there are no elements after the split
+      return { ename, month: 'november', year: undefined };
+    }
+
+    const electionCompare = (x, y) => {
+      const election1 = parseElectionName(x);
+      const election2 = parseElectionName(y);
+
+      if (election1.name > election2.name) {
+        return 1;
+      }
+      if (election1.name < election2.name) {
+        return -1;
+      }
+
+      if (election1.year > election2.year) {
+        return 1;
+      }
+      if (election1.year < election2.year) {
+        return -1;
+      }
+
+      const monthOrder = {
+        january: 0,
+        february: 1,
+        march: 2,
+        april: 3,
+        may: 4,
+        june: 5,
+        july: 6,
+        august: 7,
+        september: 8,
+        october: 9,
+        november: 10,
+        december: 11,
+      };
+
+      return monthOrder[election1.month] - monthOrder[election2.month];
```

Comment:
Similar comment to above. This is subtracting strings. Probably works without issue in JS, but I don't know. JS is wild.

Reply:
monthOrder[election.month] is not a string.

Reply:
sorry, good call